src/client/telemetry/index.ts

```diff
@@ -375,7 +375,7 @@ export interface IEventNamePropertyMapping {
         hashedName: string;
     };
     [Telemetry.HashedCellOutputMimeTypePerf]: never | undefined;
-    [Telemetry.KernelListingPerf]: never | undefined;
+    [Telemetry.KernelListingPerf]: { type: 'remote' | 'local' };
```

Comment:
Same as other, i don't think this is necessary, as we have a shared property that tells us whether user is in local or remote.

Reply:
Please remove this.